pkg/types/position.go

```diff
@@ -29,6 +29,10 @@ type PositionRisk struct {
 	LiquidationPrice fixedpoint.Value `json:"liquidationPrice"`
 }
 
+type PositionInterface interface {
```

Comment:
Call it `AnyPosition`? I don't have a better name for now. LOL

Reply:
sure